src/timeMachine/actions/queries.ts

```diff
@@ -87,26 +84,6 @@ export const setQueryResults = (
 
 let pendingCheckStatuses: CancelBox<StatusRow[][]> = null
 
-export const getOrgIDFromBuckets = (
```

Comment:
Sanity checking. This function [was put in](https://github.com/influxdata/ui/commit/45d1b2d30a5f0481a5db56bb9a31e3d07cbf2e3b) to handle the use case of querying buckets from "other" orgs. What does that mean @hoorayimhelping ? Does this use case still exist?

Reply:
Barbara [commented](https://github.com/influxdata/ui/issues/6363#issuecomment-1338625874) in the original issue that the "other" orgs is an organization that was created to receive sample data and this organization is no longer exist

Reply:
Sweet. TY!